### PR TITLE
Switch on WRIO common library

### PR DESCRIPTION
**Pull request by michbil**
Remove repeated code
Allow temporary users to use webGold functions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "should": "^6.0.3",
     "supertest": "^1.0.1",
     "winston": "^2.1.1",
-    "heapdump":"*"
+    "heapdump":"*",
+    "wriocommon"  : "michbil/WRIOOS-Common"
```
Comment (AlexeyAnshakov) [inline package.json]: must use webRunes repo

Comment (michbil) [inline package.json]: can we use my repo during active development process and switch then to webrunes?

Comment (AlexeyAnshakov) [inline package.json]: https://github.com/webRunes/webGold-WRIO-App/issues/240

Comment (michbil) [inline package.json]: @AlexeyAnshakov ok

**Pull request merged**